Flake error on PR #2129

``` 
$ flake8 scholia
scholia/api.py:72:1: W293 blank line contains whitespace
scholia/scrape/ojs.py:323:1: W293 blank line contains whitespace
```

--- a/scholia/api.py
+++ b/scholia/api.py
@@ -69,7 +69,7 @@ def entity_to_smiles(entity):
     for statement in entity['claims'].get('P233', []):
         smiles = statement['mainsnak']['datavalue']['value']
         return smiles
-    
+
     return ''
 
 

--- a/scholia/scrape/ojs.py
+++ b/scholia/scrape/ojs.py
@@ -320,7 +320,7 @@ def scrape_paper_from_url(url):
             content = _field_to_content(field)
             if content is not None and content != '':
                 return content
-        
+
         return None
 
     entry = {'url': url}

